lib/planner/search-algorithms/offline-replanning.js

```diff
@@ -162,7 +218,13 @@ module.exports = offlineReplanning = {
       if (plan.path.length > 0) {
         return null;
       } else {
-        throw new SimulatoError.PLANNER.NO_STARTING_ACTIONS('No possible actions in starting state');
+        // This will trigger if there are actions not covered, which, while a real problem
+        // shouldn't stop plans from being created. The command line output will still catch
+        // these issues and show the actions not covered.
+        const err = new SimulatoError.PLANNER.NO_STARTING_ACTIONS('No possible actions in starting state');
+        console.log(err);
```

Comment:
Should this be changed back to `throw`?

Reply:
IMO, no. I think it should be valid to be able to complete generation, even if an action is not covered. We still know the action wasn't covered, and it will show that in the output to the user, as it would when not doing replanning.

Reply:
Ok, that makes sense